ARCHITECTURE.md

```diff
@@ -177,6 +177,7 @@ Update instructions:
 |----------------------------|---------------------------------------------------------------|
 | re_build_info              | Information about the build. Use together with re_build_tools |
 | re_build_tools             | build.rs helpers for generating build info                    |
+| re_examples                | Utils for rerun example builds                                |
```

Comment:
I would expect that a crate called `re_examples` would contain examples.

Do we need this as a standalone create, or could it be part of the `re_build_examples` library? (i.e. `re_build_examples` can be both a binary and a library)

Reply:
I guess we could also just combine the library + two binaries into a single binary